app/views/users/_avatar.html.erb

```diff
@@ -1,2 +1,4 @@
 <%= stylesheet_link_tag 'users', media: 'all', 'data-turbolinks-track': 'reload' %>
-<%= image_tag avatar_user_path(user), id: "avatar-#{user.id}", height: size, width: size, class: "#{classes.join(' ')}" %>
+<%= link_to user_path(user), class: "avatar-user-link" do %>
+  <%= image_tag avatar_user_path(user), id: "avatar-#{user.id}", height: size, width: size, class: "#{classes.append('avatar-image', 'rounded-circle', 'centered-avatar').join(' ')}" %>
```

Comment:
With all these classes now in here, it would be possible to remove `classes` completely and just set it statically, wouldn't it?